### ISSUE TITLE
Atleast 2 thunderbolt 3 ports

### ISSUE THREAD

**Issue by ghost** [system76/laptop-suggestions#33]
Charging and able to connect a whole lot of devices with that port. It will look minimalistic.
Make the charger have a usb port like the one in surface chargers, its a nice feature.

**Comment by joephill11** [comment]
yes i would like at least one thunderbolt port and two would be better . definitely giving this a thumbs up
